service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageNotificationEmailService.kt

```diff
@@ -147,7 +119,7 @@ class BulletinNotificationEmailService(
         val messagesUrl = getCitizenMessagesUrl(language)
         return when (language) {
             Language.en -> """
-                <p>You have received a new eVaka bulletin/message. Read the message here: <a href="$messagesUrl">$messagesUrl</a></p>
+                <p>You have received a new eVaka Message/message. Read the message here: <a href="$messagesUrl">$messagesUrl</a></p>
```

Comment:
emacs ftw?
```suggestion
                <p>You have received a new eVaka bulletin/message. Read the message here: <a href="$messagesUrl">$messagesUrl</a></p>
```

Reply:
I actually used IDEA's search&replace, and here's the result! 😠 Should stick to Emacs.